src/semgrep_agent/utils.py

```diff
@@ -63,7 +63,10 @@ def maybe_print_debug_info(meta: "GitMeta") -> None:
 
 def render_error(error: Mapping[str, Any]) -> Sequence[str]:
     spans = error.get("spans")
-    msg = error.get("long_msg", "")
+    msg: str = error.get("long_msg", error.get("message", ""))
```

Comment:
If long_msg was removed should we just not even try to get it anymore?

Reply:
long_msg is still used in some cases (rule parsing)

Reply:
^^ yup